src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Validation/ValidationError.swift

```diff
@@ -9,3 +9,18 @@ public enum RuleValidationError: Error {
     case JSON_ENCODING_FAILED(Error?)
     case JSON_VALIDATION_RULE_SCHEMA_NOTFOUND
 }
+
+extension RuleValidationError: Equatable {
+    public static func == (lhs: RuleValidationError, rhs: RuleValidationError) -> Bool {
+        switch (lhs, rhs) {
+        case (.CBOR_DECODING_FAILED, .CBOR_DECODING_FAILED):
```

Comment:
There could be different associated errors. Can they be ignored for comparison? I ask, because I'm not sure where we need the comparison at all.

Reply:
we need this only for the unit tests, to compare if the HealthToolkit return the error we except, regardless what is inside (that is why the underlying `Error` is not compared. Do you think we should keep care of the underlying `Error`, too? Or does this is sufficient for the use case (unit test)?

Reply:
Then maybe we should add an extension to the unit test, to avoid the usage in production code. Because a future developer could think, that this is a sufficient Equatable implementation. And run into issues when she needs it in production code.

Reply:
Ah the extension in the unit test is the idea I did not had. Good one. I will move it there.